52/BlaiseCosico/pomodoro.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timedelta
+import time as time
+
+def header():
+	print(f'\nWelcome to your very own Pomodoro Timer!\n\n'
+		f'--------------INSTRUCTIONS----------------\n'
+		f'Start the Pomodoro timer\n'
+		f'Work until the Pomodoro rings\n'
+		f'Take a short break (5 minutes)\n'
+		f'Keep on working, Pomodoro after Pomodoro, until the task at hand is finished.\n'
+		f'Every 3 Pomodoros take a longer break, (15–30 minutes).\n'
+		f'Enter how many sets do you want. (Each set has 3 Pomodoros)')
```

Comment:
No need for f-strings here, also it is not "wrong". Better suited is the trible string """ ... """ which also keeps the format. f-string is normally only used if you want to format or pass variables